src/main/external-resources/renderers/Samsung-9series.conf

```diff
@@ -33,8 +33,8 @@ TranscodeAudio = WAV
 #MaxVideoWidth = 3840
 #MaxVideoHeight = 2160
 SupportedVideoBitDepths = 8,10
-H264Level41Limited = true
-#H265Level41Limited = true
+H264LevelLimit = 4.1
```

Comment:
In fact, it's depending of the resolution, 4.1 for HD and 5.1 level for UHD.

Since our DLNA is implementation is incorrect, most of the Samsung are not able to transcode without "hipcup", i choosen to set it to false before, but now 5.1 should be prefereable.